README.md

```diff
@@ -27,11 +28,17 @@ for Java, with the following changes (mandatory):
 
 * IntelliJ
   - Go to "Preferences" and them to "Coding Style" in IntelliJ.
-  - Inport [intellij-java-magicdgs-style.xml](https://github.com/magicDGS/styleguide/blob/master/intellij-java-magicdgs-style.xml)
+  - Import [intellij-java-magicdgs-style.xml](https://github.com/magicDGS/styleguide/blob/master/intellij-java-magicdgs-style.xml)
+
+* Eclipse
```

Comment:
It works, but many options are not set. Keep in mind that we need three files for eclipse, while only one of them can be imported as long as I know. The options in the other files have to be set manually.